src/pages/home-page/home-page.js

```diff
@@ -14,13 +14,17 @@ import {
   updateHomePageData
 } from '../../redux/home/home.actions';
 import { useStyles } from './home-page.styles';
+import { useCommonStyles } from '../common';
 
-const { homePageEdit } = titles;
+const { homePageTitles } = titles;
 const { IMG_URL } = config;
 
 const HomePage = () => {
+  const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed